install/src/Stack.hs

```diff
@@ -135,5 +132,5 @@ withoutStackCachedBinaries action = do
         splitPaths s =
           case dropWhile (== searchPathSeparator) s of
                       "" -> []
-                      s' -> w : words s''
+                      s' -> w : splitPaths s''
                             where (w, s'') = break (== searchPathSeparator) s'
```

Comment:
The intention of this is to split the PATH into multiple paths separated by either `:`, or `;` right? If yes, then the function: https://hackage.haskell.org/package/filepath-1.4.2.1/docs/System-FilePath-Windows.html#v:splitSearchPath looks helpful

Reply:
wow, just what i needed, many thanks!